components/atoms/Logo/index.js

```diff
@@ -0,0 +1,3 @@
+import Logo from './Logo';
+
+export default Logo;
```

Comment:
We need to discuss this with the whole team, because in PP this approach turned out to be more convenient for everyone than placing everything in the index file